src/java.base/share/classes/com/sun/crypto/provider/SunJCE.java

```diff
@@ -337,7 +345,13 @@ void putEntries() {
         attrs.clear();
         attrs.put("SupportedKeyFormats", "RAW");
 
-        if (useNativeChaCha20Cipher) {
+        if (useNativeChaCha20Cipher
+/*[IF CRIU_SUPPORT]*/
+            && !InternalCRIUSupport.isCheckpointAllowed()
+/*[ENDIF] CRIU_SUPPORT */
+            && (NativeCrypto.getVersion() >= 1)
+            && NativeCrypto.isLoaded()
```

Comment:
There's no need to call `NativeCrypto.isLoaded()`; it's implied by `NativeCrypto.getVersion() >= 1`.

Reply:
Removed